python/pyspark/sql/tests/connect/test_connect_basic.py

```diff
@@ -220,6 +220,29 @@ def test_create_global_temp_view(self):
             with self.assertRaises(_MultiThreadedRendezvous):
```

Comment:
Didn't we fix this to grpc.RPCError?

Reply:
hmmm I guess that was gone during code conflict and resolution then good fix is gone.